workshops/views.py

```diff
@@ -1,11 +1,15 @@
+import logging
+
 import yaml
 
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
 from django.core.urlresolvers import reverse
-from django.http import HttpResponseRedirect
-from django.shortcuts import render, get_object_or_404
+import django.forms
+from django.shortcuts import redirect, render, get_object_or_404
 from django.views.generic.edit import CreateView, UpdateView
 from django.db.models import Count
+from extra_views import CreateWithInlinesView, UpdateWithInlinesView, InlineFormSet
```

Comment:
On Mon, Dec 29, 2014 at 10:13:43AM -0800, W. Trevor King wrote:
> Sat, Dec 27, 2014 at 02:31:00PM -0800, Greg Wilson:
> > > +from extra_views import CreateWithInlinesView, UpdateWithInlinesView, InlineFormSet
> > 
> > What is `extra_views` ?
> 
> https://pypi.python.org/pypi/django-extra-views/, as I mentioned in
> the commit message for the “workshops.views: Edit awards in the
> person-detail edit view” commit.

This may not be the most stable dependency.  We should wait to see how
the maintainer transition [1] works out before moving forward with it.

[1]: https://github.com/AndrewIngram/django-extra-views/issues/96